src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: The Formal Mechanical Process, Section 1
+description:
+    Learn about the first section of the Formal Mechanical Process from Hegel's
+    Science of Logic
+isArticle: true
+authors: Ahilleas Rokni (2024)
+---
+
+## (a) The Formal Mechanical Process: Section 1
+
+The first section of B. The Mechanical Process that deals with the logical
+development of Mechanism is (a) The Formal Mechanical Process.
+
+The section begins with the following: “The mechanical process is the positing
+of what is contained in the Notion of mechanism, and therefore, in the first
+instance, of a _contradiction_.” (Hegel 1991, 716). The “concept” of Mechanism
```

Comment:
Remove full-stop at the end of quote? There is a full stop after the citation, perhaps that's enough?